domain/src/main/kotlin/no/nav/su/se/bakover/domain/vedtak/GjeldendeVedtaksdata.kt

```diff
@@ -16,7 +16,7 @@ import java.time.LocalDate
 
 data class GjeldendeVedtaksdata(
     // TODO Finne et bedre navn. Dette er ikke all vedtaksdata, men kun det som kan Revurderes og Reguleres
-    val periode: Periode,
+    private val periodeForTidslinje: Periode,
```

Comment:
Er dette perioden vi spesifiserte (slik det har vært tidligere, men som kanskje ikke gir mening), eller er det tidslinjen sin periode?

Reply:
Det er selve perioden for "søket" eller hva man vill kalla det. D.v.s perioden på tidslinjen. Mens `periode` er perioden på selve resultatet fra tidslinjen.